src/briefcase/integrations/xcode.py

```diff
@@ -129,14 +129,34 @@ def ensure_xcode_is_installed(
 
     :param command: The command that needs to perform the verification check.
     :param xcode_location: The location where Xcode should be installed.
-        Default is `/Applications/Xcode.app`, which is where the App Store
-        installs Xcode.
+        If not given, the location returned by `xcode-select -p` will be used.
     :param min_version: The minimum allowed version of Xcode, specified as a
         tuple of integers (e.g., (11, 2, 1)). Default: ``None``, meaning there
         is no minimum version.
     """
     # Try the direct approach. Look for the Xcode folder that is created
     # when you install from the App store.
+
+    if xcode_location is None:
+
+        try:
+            output = command.subprocess.check_output(
+                ['xcode-select', '-p'],
+                stderr=subprocess.STDOUT,
+                universal_newlines=True
+            )
```

Comment:
Have you/did you confirmed the behavior of `xcode-select` if Xcode is not installed? I've got a VM installation I can use to confirm this, but if you've already done this check, then I won't bother. My concern is that there may be an Xcode-select stub that triggers the App Store if Xcode isn't there.

Reply:
Yes, I've checked this in a VM myself. It errors out with the following message:

> xcode-select: error: unable to get active developer directory, use `sudo xcode-select --switch path/to/Xcode.app` to set one (or see `man xcode-select`)